etc/kayobe/inventory/host_vars/ethsw-b17-u40-41

```diff
@@ -60,19 +60,26 @@ switch_interface_config:
   Te1/1/3/4:
     config: "{{ switch_interface_config_all }}"
   Hu1/4/7:
-    description: ethsw-b16-u40-41-trunk1
+    description: ethsw-b17-u40-41-trunk1
```

Comment:
In which case should we rename the ports at the other end of the lag?  Both are named the same.